fleet_manager:

  main_timer_rate: 100.0 # [Hz]
  feedback_timer_rate: 100.0 # [Hz]
  no_message_timeout: 5.0 # [s]

  planners:
    # the list of "names" of the loaded plugins
    # each plugin's properties are defined in 'plugins.yaml'
    planner_names: [
      "WaypointPlanner",
      "CoveragePlanner",
      "AutonomyTestPlanner",
    ]

    # define "running instances" of plugins, there can be more than one for each
    WaypointPlanner:
      address: "iroc_fleet_manager/WaypointPlanner" 
      name_space: "waypoint_planner" 

    CoveragePlanner:
      address: "iroc_fleet_manager/CoveragePlanner" 
      name_space: "coverage_planner" 

    AutonomyTestPlanner:
      address: "iroc_fleet_manager/AutonomyTestPlanner" 
      name_space: "autonomy_test_planner" 

    # rate to update the common handlers
    update_timer_rate: 1 # [Hz]
